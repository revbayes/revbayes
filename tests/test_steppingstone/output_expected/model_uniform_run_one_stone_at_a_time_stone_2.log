Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21318.64	-21282.24	-36.39926	0.5823748	0.170625	0.207669	0.223602	0.106295	0.149705	0.272919	0.475682	0.142732	0.245006	0.0944096	0.36703	0.166413	0.188506	0.279597	0.24467	0.13663	0.240999	0.183608	0.246636	0.351343	0.342348	0.234207	0.442078	0.106288	0.0543757	0.00113704	0.136554	0.101541	0.0642406	0.347176	0.328512	0.122558	0.0820228	0.00870705	0.0665103	0.203455	0.0560237	0.0699039	0.0238113	0.00989447	0.260587	0.190286	0.463901	0.0232973	0.284595	0.0590038	0.0244203	0.569689	0.0389951	0.36752	0.336847	0.0831614	0.212472	0.3399655	8.210191
10	-21311.71	-21284.45	-27.26376	0.699525	0.111955	0.156826	0.174299	0.0902683	0.157669	0.219751	0.43536	0.179046	0.179352	0.144559	0.227374	0.19052	0.181012	0.314195	0.263574	0.0866962	0.146746	0.210092	0.258891	0.230692	0.199556	0.165739	0.659017	0.121045	0.0552211	0.000527675	0.141451	0.067116	0.0499223	0.367001	0.286664	0.0951868	0.0820301	0.0133839	0.0868895	0.203455	0.00818666	0.0467213	0.0435794	0.00723159	0.190875	0.169423	0.31988	0.0318227	0.300838	0.0592523	0.0248185	0.55461	0.0286585	0.380628	0.321888	0.0762154	0.221268	0.3577695	7.338983
