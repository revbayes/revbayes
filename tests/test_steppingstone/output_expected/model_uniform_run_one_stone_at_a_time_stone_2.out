state	power	likelihood
4	0.32768	-21298.7
5	0.32768	-21314.2
6	0.32768	-21314
7	0.32768	-21320
8	0.32768	-21291.1
9	0.32768	-21283.1
10	0.32768	-21293.7
